File: __tests__/fixtures/output/javascript/jquery/query.js
const settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value",
  "method": "GET",
  "headers": {}
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: __tests__/__fixtures__/output/ruby/native/query.rb
require 'uri'
require 'net/http'
require 'openssl'

url = URI("https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body
